kube/Dockerfile.labextensions

```diff
@@ -1,4 +1,4 @@
-FROM cdoutrix/vcdat:core
+FROM cdoutrix/vcdat_core:latest
 
 RUN jupyter labextension install @jupyter-widgets/jupyterlab-manager && jupyter labextension install @jupyter-widgets/jupyterlab-sidecar && jupyter labextension install @jupyterlab/github && pip install jupyterlab_github && jupyter serverextension enable --sys-prefix jupyterlab_github && pip install nbgitpuller && jupyter serverextension enable --py nbgitpuller --sys-prefix && jupyter labextension install @jupyterlab/hub-extension && rm -rf node_modules && rm -rf ~/.cache/pip
```

Comment:
Need to add the tutorial manager extension I made: && jupyter labextension install tutorial-manager-extension

Reply:
Done and merged into single docker file.